socobo-inventory.html

```diff
@@ -18,11 +18,22 @@
     </style>
     <div>
       <socobo-inventory-create id="inventoryCreateDialog"></socobo-inventory-create>
+      
+      <paper-dialog id="confirmationDialog" modal>
+        <h2>Remove item</h2>
+        <span>Are you sure you want to remove this item?</span>
+        <div class="buttons">
+          <paper-button dialog-dismiss>Cancel</paper-button>
+          <paper-button dialog-confirm on-click="_confirmDelete">Ok</paper-button>
+        </div>
+      </paper-dialog>
```

Comment:
This is the confirmation dialog. I am unsure whether this limited functionality (essentially just using the `paper-dialog` warrants a whole Polymer element